include/vcpkg/base/messages.h

```diff
@@ -1372,7 +1372,7 @@ namespace vcpkg
         (msg::count, msg::elapsed, msg::value),
         "{value} may be either a 'vendor' like 'Azure' or 'NuGet', or a file path like C:\\example or /usr/example",
         "Restored {count} package(s) from {value} in {elapsed}. Use --debug to see more details.");
-    DECLARE_MESSAGE(ResultsHeader, (), "Displayed before a list of installation results.", "RESULTS");
+    DECLARE_MESSAGE(ResultsHeader, (), "Displayed before a list of installation results.", "\nRESULTS");
```

Comment:
We don't put formatting concerns like extra blank lines in localized messages, specifically because that's not a thing that translators should be fighting with. If you want a blank line in a place print it explicitly.

Reply:
Done.